pkg/test/framework/components/namespace/namespace.go

```diff
@@ -53,6 +53,16 @@ func ClaimOrFail(t test.Failer, ctx resource.Context, name string) Instance {
 
 // New creates a new Namespace in all clusters.
 func New(ctx resource.Context, nsConfig Config) (i Instance, err error) {
+	settings, err := resource.SettingsFromCommandLine(ctx.Settings().TestID)
+	if err != nil {
+		return nil, err
+	}
+	// Overwrite the default namespace label (istio-injection=enabled)
+	// with istio.io/rev=XXX. If a revision label is already provided,
+	// the label will remain as is.
+	if nsConfig.Revision == "" && settings.Revision != "" {
```

Comment:
the extra ` && settings.Revision != "" ` has no effect

Reply:
Removed.